client/src/actions/actions.js

```diff
@@ -0,0 +1 @@
+export const SET_AUTHENTICATED = 'SET_AUTHENTICATED';
```

Comment:
move all constants to a constants file, so that when you want to change a constant you know where to look for it.
Also doesn't make sense for a reducer to import constants from actions :)

Reply:
**actions.js** is that constants file. It will have all the action constants that will be imported by reducers/components to dispatch actions.

Reply:
hmm, in that case rename the file, it is misleading . and usually we have constants at the root or in constants folder.

Reply:
I've renamed **actions.js** to **actionTypes.js** and pushed the changed. Let me know if it is okay now.